src/command_modules/azure-cli-acs/HISTORY.rst

```diff
@@ -7,6 +7,7 @@ unreleased
 * correct preview regions
 * format default dns_name_prefix properly
 * optimize acs command output
+* improve documentions
```

Comment:
History entry not required/desired for docs improvements. Only customer facing behavioral changes.